### DISCUSSION TITLE
Duda sobre características para determinar el buyer.

### DISCUSSION THREAD

**Discussion by MauricioGongoraLoria** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#48] [Q&A]
El material de apoyo hace mucha referencia sobre el apartado técnico, sin embargo el primer tema se enfoco a determinar el buyer para eso hay que identificar primero a nuestro usuario según entendí. Ahora bien las demandas que se pueden hacer dependen de la rama del derecho y la forma en la cual se tiene puede representar una persona,  los requisitos son diferentes si la persona es física o moral y dependiendo el asunto hay derecho laboral, fiscal, civil, penal, etc. y no todas se pueden hacer de manera remota por así decirlo; dado lao anterior ¿es necesario entonces considerar que se tiene algún tipo de infraestructura, además de la existencia de un contrato de servicios, de igual manera el tipo de tratamiento que se le dará por ejemplo a los archivos en materia fiscal, (firma electrónica)? o solo es considerar a la "demanda" como un solo producto genérico...

**Comment by Ingrzs** [comment]
El buyer es tu cliente unico, todo lo demas que describiste seria tu publicación objetivo (general) ... En live del lunes y miércoles explican esa parte

**Reply by MauricioGongoraLoria** [reply]
¿Cómo determinaste tu buyer??

**Reply by Ingrzs** [reply]
Cuando descargaste el archivo práctica 2 el PDF,trae predifinido unos datos menor edad educación secundaria, yo ocupe ese, pero igual puedes cambiar ese y ponerle otro, y en base eso ir recopilando datos de acuerdo a la descripción de la práctica....en base a eso sale el público objetivo....

**Reply by GloriaCoralCerecedo** [reply]
El buyer person y el público objetivo, ¡serían las personas que van a usar a aplicación de los abogados?

**Reply by MauricioGongoraLoria** [reply]
Entiendo que el buyer es la persona que va a utilizar la aplicación, pero a lo que voy, cuando tu desarrollas un producto o en este caso una herramienta,  tienes que determinar quien te lo va a comprar (o usar) y como se lo harás atractivo, lo haces segmentando y conociendo el producto o servicio que vas a vender hasta llegar al perfil que consideras mas apegado a lo que vas determinando (buyer persona) para con esto; saber como llegar a el;  es diferente llevar una demanda de pensión alimenticia que tiene que haber visitas, comparecencias y todo ese rollo (lo tiene que hacer el demandado)  a una demanda por no haber presentado declaraciones fiscales que algunas veces se puede hacer en linea, para la primera la aplicación por lo menos el despacho debería tener presencia en el domicilio del demandante o demandado, para la otra muy probablemente no.  A lo que voy, que tan profundo debe ser el análisis de esas variables  que conlleva un proceso legal para este ejercicio en particular, o solo se va a ver la demanda como un proceso tipo compra de mercado libre que ahí vas viendo como va tu compra, tu pedido, etc.

**Reply by Ingrzs** [reply]
Tan profundo como tu lo desees o hasta donde consideres tu proyecto , pero considera las partes que el cliente quiere, por eso en la descripción del tema dice el cliente acepta propuestas,  es decir tu le añades cuanta mejoria quieres, y ademas de eso cuando haces el diagrama de información o el wireframe se ve todo lo que haria tu buyer y tu público objetivo en el sistema (hasta te salen mas cosas ) .
Y @GloriaCoralCerecedo asi es,tu sistema lo usará tu cliente, sus clientes , considera los pasos que haria cada uno en el sistema desde el inicio de crear una demanda hasta terminarla.

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias!

**Comment by DanyVeneno** [comment]
¡Hey! Mauricio si te quieres ir más profundo que te lo recomiendo por que asi tienes una perspectiva más amplia a la hora de ya estar en un proyecto real te recomiendo tomes un par de talleres uno es en Hubspot(inboundMarketing) y otro en Rock University(Estrategia de Marketing) create una cuenta en estas dos plataformas y busca los talleres solo le vas a invertir tiempo 5hrAprox. Y lo vas a poder complementar muy bien con lo que nos explico el MCRodrigo y al rato hay que ver el feedback. Saludos buen tarde

**Reply by MauricioGongoraLoria** [reply]
Muchas gracias, saludos!
